bower_components/aura/lib/ext/mediator.js

```diff
@@ -92,7 +91,6 @@ define('aura/ext/mediator', function () {
        * @param {Object} payload    Payload emitted
        */
       app.sandbox.emit = function () {
-        if (this.stopped) { return; }
```

Comment:
if stopped is true then the sandbox was stopped before ... maybe someone closes the data-navigation before? i think we have "removed" all lost components but i dont want to get into troubles with that because we dont know if there will be some lost components in the future ....